common/src/nostr/mod.rs

```diff
@@ -89,7 +89,7 @@ impl NostrClient {
                             let rumor = self.client.unwrap_gift_wrap(&event).await?.rumor;
                             if rumor.kind == Kind::PrivateDirectMessage {
                                 let result = serde_json::from_str::<T>(&rumor.content)
-                                    .map_err(|e| anyhow::Error::new(e));
+                                    .map_err(anyhow::Error::new);
```

Comment:
redundant closure
for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#redundant_closure
`#[warn(clippy::redundant_closure)]` on by default